src/aiod/authorisation/authorisation.py

```diff
@@ -0,0 +1,80 @@
+import os
+from keycloak import KeycloakOpenID
+
+from aiod.config.settings import server_url, client_id, realm
+
+keycloak_openid = KeycloakOpenID(
+    server_url=server_url,
+    client_id=client_id,
+    realm_name=realm,
+)
+
+
+def login(username: str, password: str) -> None:
```

Comment:
Note: this is fine for now, but we need to open an issue to allow username/password to be read from the environment and/or file.

Reply:
I agree, that's what I had in mind as well. :)

Reply:
An update here. Me and Rafa reviewed keycloak realm and client configuration and we realised that we can configure a refresh token (or an offline token - I still have to double check the right one) to have long duration. Then we can safely store it to a file instead of the user credentials. I'll mention this in a new issue.